src/Host/Client/Impl/Host/LocalBrokerClient.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Microsoft.R.Host.Client.Host {
     internal sealed class LocalBrokerClient : BrokerClient {
-        private const string RHostBrokerExe = "Microsoft.R.Host.Broker.exe";
+        private const string RHostBrokerExe = "Microsoft.R.Host.Broker.Windows.exe";
```

Comment:
It's the same broker, just different name  cause xplat assembly is `Microsoft.R.Host.Broker.dll`